src/tests/components/signIn.test.tsx

```diff
@@ -0,0 +1,53 @@
+
+import { render, screen, fireEvent } from "@testing-library/react";
+import { BrowserRouter } from "react-router-dom";
+import SignIn from '../../components/SignIn';
+import axios from 'axios';
+import MockAdapter from 'axios-mock-adapter';
+
+
+describe('Sign in component', () => {
+  beforeEach(()=>{
+    render(<BrowserRouter>
+      <SignIn />
+      </BrowserRouter>);
+  })
+
+  it('renders without crashing', () => {
+    const signIn = render(<BrowserRouter> <SignIn />  </BrowserRouter>);
+    expect(signIn).toBeDefined();
+  });
+
+  it ('renders correct inputs', () =>{
+    const inputEmail = screen.getByRole('textbox', {name: 'EMAIL'});
+
+    expect(inputEmail).toBeDefined();
+  });
+
+  it('renders title', () => {
+    const title = screen.getByText('TARGET MVD', {selector: 'h1'})
```

Comment:
With this change, you will find your text no matter if it's uppercased or lowercased. WDYT?
```suggestion
    const title = screen.getByText(/TARGET MVD/i, {selector: 'h1'})
```